documents/payment.md

```diff
@@ -598,7 +598,192 @@ Doc reference [doc](https://razorpay.com/docs/payments/payment-methods/cards/aut
   "razorpay_payment_id": "pay_JWaNvYmrx75sXo"
 }
 ```
+-------------------------------------------------------------------------------------------------------
+### Create Payment Json (Third party validation)
+
+```java
+
+JSONObject paymentRequest = new JSONObject();
+paymentRequest.put("amount",500);
+paymentRequest.put("currency","INR");
+paymentRequest.put("email", "gaurav.kumar@example.com");
+paymentRequest.put("contact", "9123456789");
+paymentRequest.put("order_id", "order_JZluwjknyWdhnU");
+paymentRequest.put("method", "netbanking");
+paymentRequest.put("bank", "HDFC");
+              
+Payment payment = instance.payments.createJsonPayment(paymentRequest);
+```
+
+**Parameters:**
+| Name        | Type    | Description                          |
+|-------------|---------|--------------------------------------|
+| amount*          | integer | Amount of the order to be paid  |
+| currency*   | string  | The currency of the payment (defaults to INR)                                  |
+| order_id*        | string  | The unique identifier of the order created. |
+| email*        | string      | Email of the customer                       |
+| contact*      | string      | Contact number of the customer              |
+| method*      | string  | Possible value is `netbanking` |
+| bank      | string      | The customer's bank code.For example, `HDFC`.|
```

Comment:
bank is mandatory

Reply:
[added](https://github.com/razorpay/razorpay-java/blob/third_party_validation_payment/documents/payment.md?plain=1#L625)